lib/rihanna/migration/upgrade.ex

```diff
@@ -116,6 +127,17 @@ defmodule Rihanna.Migration.Upgrade do
       """,
       """
       CREATE INDEX IF NOT EXISTS rihanna_jobs_enqueued_at_id ON rihanna_jobs (enqueued_at ASC, id ASC);
+      """,
+      """
+      ALTER TABLE #{table_name} ADD COLUMN priority integer NOT NULL DEFAULT 19;
```

Comment:
This migration is so far idempotent, so for all the other `ADD COLUMN` statements they're wrapped in `BEGIN ... EXCEPTION ... END` blocks.

I.e. this should be:

```suggestion
      DO $$
          BEGIN
              ALTER TABLE #{table_name} ADD COLUMN priority integer NOT NULL DEFAULT 19;
          EXCEPTION
              WHEN duplicate_column THEN
              RAISE NOTICE 'column already exists in #{table_name}.';
          END;
      $$
```